### DISCUSSION TITLE
How to get the key of the selected value from Autocomplete?

### DISCUSSION THREAD

**Discussion by anesask** [mantinedev/mantine#2566] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

I am trying to retrieve id or key value from selected option from Autocomplete element.
I am getting value, but cannot manage to get key. 
Thanks in advance.

### What version of @mantine/hooks page do you have in package.json?

5.4.0

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/embed/suspicious-lederberg-1vee76?fontsize=14&hidenavigation=1&theme=dark

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by rtivital** [comment]
https://codesandbox.io/s/nice-booth-t78l0y?file=/src/App.tsx

**Reply by anesask** [reply]
> https://codesandbox.io/s/nice-booth-t78l0y?file=/src/App.tsx

Somehow I can't make it work with my Autocompleter element.
